### ISSUE TITLE
Disabled field values too light in Firefox

### ISSUE THREAD

**Issue by Stanton** [jadu/pulsar#1098]
**Describe the bug**
Values with disabled fields in Firefox is adopting an incorrect text colour from the browser's forms.css default stylesheet.

**Expected behavior**
The text should use the standard base text colour and meet WCAG AA contrast when set against the disabled field background.

**Screenshots**
![Screenshot 2019-12-02 at 13 42 17](https://user-images.githubusercontent.com/18653/69964261-f22ac680-1509-11ea-9978-6bd7c0787c93.png)

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser: Firefox
 - Version: Observed in 70.0.1

**Comment by Stanton** [comment]
Added a [fix](https://github.com/jadu/pulsar/commit/fe7e3a8926f579203f164784468383a15790ee7c) to the CMS `flatcap` branch which will resolve this issue.

**Before**
![Screenshot 2019-12-02 at 14 57 21](https://user-images.githubusercontent.com/18653/69972101-ea265300-1518-11ea-80d9-918adf0ae056.png)

**After**
![Screenshot 2019-12-02 at 15 03 59](https://user-images.githubusercontent.com/18653/69972110-eeeb0700-1518-11ea-9105-2e0f0ed5070c.png)

**Comment by jackbentley** [comment]
I feel like the value could cause confusion and be accidentally interpreted as editable - it looks like it is?

Would it fall under the category of not needing to be contrast compliant? https://ux.stackexchange.com/questions/80881/do-disabled-buttons-still-need-to-be-contrast-compliant-for-accessibility

**Comment by Stanton** [comment]
![field](https://user-images.githubusercontent.com/18653/69976358-c535de00-1520-11ea-93a9-6345e194aad4.gif)

The background colour reduces the overall contrast, coupled with the other affordances of the element, I think the text colour change here is better than the original light grey.

**Comment by jackbentley** [comment]
Agreed. I think the light grey isn't useful to anyone (accessibility or not).

I thought this was a PR from the before/after screenshot. In that image, the after text looks like a black. In the gif it looks better as it's a dark grey - still readable but still obviously not enabled 👍
